packages/smarthr-ui/src/components/TabBar/TabBar.stories.tsx

```diff
@@ -32,10 +33,15 @@ const Template: StoryFn = ({ subid, ...props }) => (
     <TabItem id={`border-${subid}-1`} onClick={action('clicked')} selected>
       基本情報
     </TabItem>
-    <TabItem id={`border-${subid}-2`} onClick={action('clicked')}>
+    <TabItem id={`border-${subid}-2`} onClick={action('clicked')} suffix={<Badge count={7} />}>
       コメント
     </TabItem>
-    <TabItem id={`border-${subid}-3`} onClick={action('clicked')} disabled>
+    <TabItem
+      id={`border-${subid}-3`}
+      onClick={action('clicked')}
+      disabled
+      disabledDetail={{ message: 'この機能は使用できません。' }}
```

Comment:
```suggestion
      disabledDetail={{ message: 'このタブは使用できません。' }}
```

disabledDetail のメッセージが「この機能は使用できません。」になってる理由ってありますか？（ピュア）タブアイテム=機能っていうメンタリティがあまり沸かなかったので、自分ならマテオネで「タブ」にすると思いました。明確な理由とメリットがあるならOKです！

Reply:
利用者が「タブ」を認識しているとは限らないので、実際に表すオブジェクトを指すと考え、この Story では「機能」なのかな、と思って書きました!